src/components/Comments.jsx

```diff
@@ -18,7 +18,7 @@ const Comments = ({ article }) => {
         setIsLoading(false);
         setIsError(true);
       });
-  }, [article.article_id]);
+  }, [article.article_id, newComment]);
```

Comment:
⚠️ watch out!

Whenever `newComment` updates this useEffect will run

This means **every time a user types in the comment box a new request will be made** 😱

Reply:
I will have to refactor that, completly overlooked, it is actually making a request at every character change